scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -480,16 +480,13 @@ def build_phase(self, sharedlib_only=False, model_only=False):
 """
 echo Insta pass
 echo SUCCESSFUL TERMINATION > {}/cpl.log.$LID
-cp {}/scripts/tests/cpl.hi1.nc.test {}/{}.cpl.hi.0.nc.base
+cp {}/scripts/tests/cpl.hi1.nc.test {}/{}.cpl.hi.0.nc
 cp {}/scripts/tests/cpl.hi2.nc.test {}/{}.cpl.hi.0.nc.rest
 """.format(rundir, cimeroot, rundir, case, cimeroot, rundir, case)
         self._set_script(script)
         super(TESTTESTDIFF, self).build_phase(sharedlib_only=sharedlib_only,
                                               model_only=model_only)
 
-    def run_indv(self, suffix=None, st_archive=False ):
```

Comment:
@jedwards4b this is probably what caused the copy fail above (this run_indv changed its defaults). Looks like you found a way around it, so this is a good change.